<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectPlaceSettingDlg</class>
 <widget class="QDialog" name="ObjectPlaceSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>New Polygon</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gruGeneral">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="title">
      <string>General</string>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>StartLevel</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spnStartLevel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtName_ko">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>55</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>55</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>EndLevel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbObject">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Object</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Name_ko </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtName">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>55</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spnEndLevel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>62</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ShapeType</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbShapeType">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gruIcon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Icon</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>IconSize</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>IconPath</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtIconPath">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>271</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnIconPath">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spnIconSize">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
     </widget>
     <widget class="QSlider" name="sldIconSize">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>256</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gruLine">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Line</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QDoubleSpinBox" name="spnLineThick">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>LineColor</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>LineThick</string>
      </property>
     </widget>
     <widget class="QSlider" name="sldLineThick">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Opacity</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spnLineOpacity">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string> %</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLineColor">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>18</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gruFill">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>Fill</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>21</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>FillColor</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Opacity</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>FillTexture</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtTexturePath">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>271</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTexturePath">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spnFillOpacity">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string> %</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFillColor">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>18</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gruPreview">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>160</height>
      </size>
     </property>
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>7</number>
      </property>
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbObject</tabstop>
  <tabstop>edtName_ko</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>cmbShapeType</tabstop>
  <tabstop>spnStartLevel</tabstop>
  <tabstop>spnEndLevel</tabstop>
  <tabstop>gruIcon</tabstop>
  <tabstop>edtIconPath</tabstop>
  <tabstop>btnIconPath</tabstop>
  <tabstop>spnIconSize</tabstop>
  <tabstop>sldIconSize</tabstop>
  <tabstop>gruLine</tabstop>
  <tabstop>spnLineOpacity</tabstop>
  <tabstop>spnLineThick</tabstop>
  <tabstop>sldLineThick</tabstop>
  <tabstop>gruFill</tabstop>
  <tabstop>spnFillOpacity</tabstop>
  <tabstop>edtTexturePath</tabstop>
  <tabstop>btnTexturePath</tabstop>
  <tabstop>graphicsView</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>applyButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
